app/assets/stylesheets/recipes.scss

```diff
@@ -28,6 +28,63 @@
     border: 0;
     margin-top: 1rem;
   }
+
+  input#recipe_archived {
```

Comment:
The css linter doesn't like id selectors. I'll defer to you if you want to change it.